index.js

```diff
@@ -42,6 +42,11 @@ class ClinicDoctor extends events.EventEmitter {
     this.detectPort = detectPort
     this.debug = debug
     this.path = dest
+    this.issue = null
+  }
+
+  getIssue () {
+    return this.issue
   }
```

Comment:
Not sure this separate function is useful given that it just returns a public property. Someone could just grab `this.issue` directly

Reply:
Oh yeah good shout, I didn't see that. Should issue be private or do you reckon it's alright to have it as a public member?

Reply:
> Oh yeah good shout, I didn't see that. Should issue be private or do you reckon it's alright to have it as a public member?

I'm running into the unexpected '#' problem, I'm going to go with leaving it as a public member